numba/cuda/cudadrv/devicearray.py

```diff
@@ -331,6 +331,13 @@ def view(self, dtype):
             gpu_data=self.gpu_data,
             )
 
+    @property
+    def nbytes(self):
+        if hasattr(self, 'alloc_size'):
```

Comment:
Is it possible to create a device array through documented/public API means which doesn't have an `alloc_size` attr?

Reply:
switch to size * dtype.itemsize